python/lsst/dax/apdb/sql/apdbSql.py

```diff
@@ -1230,3 +1230,16 @@ def _add_spatial_index(self, df: pandas.DataFrame) -> pandas.DataFrame:
         df = df.copy()
         df[self.config.htm_index_column] = htm_index
         return df
+
+    def _fix_timestamps(self, df: pandas.DataFrame) -> pandas.DataFrame:
+        """Update timestamp columns to be naive datetime type.
+
+        AP pipeline code expects DataFrames to contain naive datetime columns,
+        while Postgres queries return timezone-aware type. This method converts
+        those columns to naive datetime in UTC timezone.
+        """
+        # Find all columns with aware timestamps.
+        columns = [column for column, dtype in df.dtypes.items() if isinstance(dtype, pandas.DatetimeTZDtype)]
+        for column in columns:
+            df[column] = df[column].dt.tz_convert(None)
```

Comment:
I find the Codecov warning here a bit disturbing. That would seem to imply that `columns` is empty.

Reply:
This code is currently only used for Postgres backend, I need to extend Github action to enable Postgres testing, always forgetting.